packages/cosmic-swingset/test/test-home.js

```diff
@@ -120,9 +132,26 @@ test('home.wallet', async t => {
       `wallet knows about the Zoe invite issuer`,
     );
     const invitePurse = await E(wallet).getPurse('Default Zoe invite purse');
+    const zoeInviteBrand = await E(invitePurse).getAllegedBrand();
     t.deepEquals(
```

Comment:
Not sure if you want deepEquals here.  I think we can depend on the identity of the brand, so `t.equals` should suffice.  If I'm wrong, let's document the reasons.

Reply:
very good catch, thanks! I should use `equals`